data/kestrel.txt

```diff
@@ -317,16 +323,35 @@ ship "Kestrel"
 	description "Several years ago, while visiting a sweat lodge in search of spiritual renewal, Tarazed's chief ship designer fell into a trance and journeyed to a reality separate from our own. There he saw visions of a strange and elegant starship. Returning to his work with renewed purpose, he spent the next seven years making the Kestrel a reality."
 
 
-
-ship "Kestrel" "Kestrel (More Weapons)"
-	add attributes
-		"weapon capacity" 20
-
 ship "Kestrel" "Kestrel (More Engines)"
 	add attributes
-		"engine capacity" 20
+		"drag" -1.0
+		"fuel capacity" 100
+		"engine capacity" 30
 
 ship "Kestrel" "Kestrel (More Shields)"
 	add attributes
-		"shields" 2700
-		"hull" 1300
+		"shields" 3000
+		"hull" 1500
+
+ship "Kestrel" "Kestrel (More Weapons)"
+	add attributes
+		"weapon capacity" 40
+	gun -75 57 "Torpedo Launcher"
+	gun 75 57 "Torpedo Launcher"
+	gun -75 57
+	gun 75 57
+	gun -75 57
+	gun 75 57
+	gun -53 61 "Particle Cannon"
+	gun 53 61 "Particle Cannon"
+	gun -53 61
+	gun 53 61
+	gun -53 61
+	gun 53 61
+	gun -31 66 "Particle Cannon"
+	gun 31 66 "Particle Cannon"
+	gun -31 66
+	gun 31 66
+	gun -31 66
+	gun 31 66
```

Comment:
You also need to add the turret coordinates here, as changing any single weapon coordinate with a variant overwrites all previous weapon coordinates.
I'd also reorder these gun ports so that it goes 
```
-75
-75
-75
75
75
75
...
```
instead of 
```
-75
75
-75
75
...
```

so that it looks like this 
![image](https://user-images.githubusercontent.com/17688683/48529576-b8851000-e861-11e8-9ec7-f985ba994ccf.png)
instead of this
![image](https://user-images.githubusercontent.com/17688683/48529584-c2a70e80-e861-11e8-9f55-75404a59eadb.png)

Reply:
its... beautiful 😍

Reply:
it's... beautiful! 😍

Reply:
Done.